dspace/config/spring/api/discovery.xml

```diff
@@ -289,6 +301,219 @@
+                    <list>
+                        <value>dc.title</value>
+                        <value>dc.contributor.author</value>
+                        <value>dc.creator</value>
+                        <value>dc.subject</value>
+                    </list>
+                </property>
+                <!--The minimum number of matching terms across the metadata fields above before an item is found as related -->
+                <property name="minTermFrequency" value="5"/>
+                <!--The maximum number of related items displayed-->
+                <property name="max" value="3"/>
+                <!--The minimum word length below which words will be ignored-->
+                <property name="minWordLength" value="5"/>
+            </bean>
+        </property>
+        <!-- When true a "did you mean" example will be displayed, value can be true or false -->
+        <property name="spellCheckEnabled" value="true"/>
+    </bean>
+	
+	<!--The workflow configuration settings for discovery -->
+	<bean id="workflowConfiguration"
+		class="org.dspace.discovery.configuration.DiscoveryConfiguration"
+		scope="prototype">
+		<property name="id" value="workflow" />
+		<!--Which sidebar facets are to be displayed -->
+		<property name="sidebarFacets">
+			<list>
+				<ref bean="searchFilterObjectNamedType" />			
+				<ref bean="searchFilterType" />
+				<ref bean="searchFilterIssued" />
+				<ref bean="searchFilterSubmitter" />
+			</list>
+		</property>
+		<!--The search filters which can be used on the discovery search page -->
+		<property name="searchFilters">
+			<list>
+				<ref bean="searchFilterObjectNamedType" />			
+				<ref bean="searchFilterType" />
+				<ref bean="searchFilterIssued" />
+				<ref bean="searchFilterSubmitter" />
+			</list>
+		</property>
+        <!--The sort filters for the discovery search-->
+        <property name="searchSortConfiguration">
+            <bean class="org.dspace.discovery.configuration.DiscoverySortConfiguration">
+                <!--<property name="defaultSort" ref="sortDateIssued"/>-->
+                <!--DefaultSortOrder can either be desc or asc (desc is default)-->
+                <property name="defaultSortOrder" value="desc"/>
+                <property name="sortFields">
+                    <list>
+                        <ref bean="sortTitle" />
+                        <ref bean="sortDateIssued" />
+                    </list>
+                </property>
+            </bean>
+        </property>
+		<!--Any default filter queries, these filter queries will be used for all 
+			queries done by discovery for this configuration -->
+		<property name="defaultFilterQueries">
+			<list>
+				<!--Only find PoolTask and ClaimedTask -->
+				<value>search.resourcetype:[10 TO 11]</value>
+			</list>
+		</property>
+        <!--The configuration for the recent submissions-->
+        <property name="recentSubmissionConfiguration">
```

Comment:
Can this property not be left out, I don't think we need the recent submissions configuration for workflow items.

Reply:
removed, see https://github.com/4Science/DSpace/pull/65/commits/cc11f4955677eaedfae70c54eb08e940bb6067fe